js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedAssetRow.tsx

```diff
@@ -67,13 +68,18 @@ export const VirtualizedAssetRow = (props: AssetRowProps) => {
   } = props;
 
   const liveData = useLiveDataOrLatestMaterializationDebounced(path, type);
-  const linkUrl = assetDetailsPathForKey(
+  let linkUrl = assetDetailsPathForKey(
     {path},
     {
       view: type === 'folder' ? 'folder' : undefined,
     },
   );
 
+  if (type === 'folder') {
+    // Forward filters
+    linkUrl += getAssetFilterStateQueryString();
```

Comment:
Is `linkUrl` just a path prior to this concatenation? Is there already a `?` applied to it?

Reply:
yeah folders already get a `?`